doc/classes/Time.xml

```diff
@@ -180,7 +180,7 @@
 		<method name="get_unix_time_from_system" qualifiers="const">
 			<return type="float" />
 			<description>
-				Returns the current Unix timestamp in seconds based on the system time in UTC. This method is implemented by the operating system and always returns the time in UTC.
+				Returns the current Unix timestamp in seconds based on the system time. The Unix timestamp is the number of seconds passed since 00:00:00 UTC on 1 January 1970, the [url=https://en.wikipedia.org/wiki/Unix_time]Unix epoch[/url]. It is always in UTC.
```

Comment:
This is not correct. The timestamp does not need to be UTC.

Most of the `Time` class works with "the same timezone", so you can give it a timestamp in any time zone. Saying that it is always in UTC is not consistent with the rest of the Time class which allows non-UTC timestamps.

Reply:
I meant that the Unix epoch is based on UTC (00:00:00 UTC on 1 January 1970), and since this method returns seconds passed since that time, I thought it would mean it's in UTC as well. 

Should I just remove the last phrase "It is always in UTC." ?

Reply:
You can change it to this:

> Returns the current Unix timestamp in seconds based on the system time in UTC. The Unix timestamp is the number of seconds passed since 1970-01-01 at 00:00:00, the [url=https://en.wikipedia.org/wiki/Unix_time]Unix epoch[/url].

Reply:
Thank you for the quick reply. I'm slightly confused about keeping "always returns the time in UTC" because it feels like we wouldn't be addressing the problem raised in the original issue. 

> This method returns a time interval. It doesn't make sense to state "returns the time in UTC". A time interval is independent of any time zone.

Sorry for the confusion.

Reply:
Thanks for your PR trying to solve this. There is some misunderstanding in the issue, I commented there.

Reply:
Oh I see, thanks for the clarification there.